Formula/wine.rb

```diff
@@ -427,20 +421,14 @@ def install
       ENV["ac_cv_lib_soname_#{dep}"] = (libexec/"lib/lib#{dep}.dylib").realpath
     end
 
-    if build.with? "win64"
-      args64 = ["--prefix=#{prefix}"] + depflags
-      args64 << "--enable-win64"
-      args64 << "--without-x" if build.without? "x11"
+    args64 = ["--prefix=#{prefix}", "--enable-win64", "--without-x"] + depflags
 
-      mkdir "wine-64-build" do
-        system "../configure", *args64
-        system "make", "install"
-      end
+    mkdir "wine-64-build" do
+      system "../configure", *args64
```

Comment:
No need for the `args64` variable anymore:
`system "../configure", *depflags, "--prefix=#{prefix}", "--enable-win64", "--without-x"`
(with appropriate line wrapping)